framework/midi/midievent.h

```diff
@@ -23,6 +23,10 @@
 #include <set>
 #include "audio/midi/event.h"
 
+#ifndef UNUSED
+#define UNUSED(x) (void)x;
```

Comment:
Why not using `Q_UNUSED()`?

Reply:
We try not to use Qt where it is not used.

Reply:
and reinventing the wheel when not needed?

Reply:
We really want to limit the use of Qt

Reply:
Why?

Reply:
We want to create a platform that is Qt independent and has an interface built using Qt.
This will give us the freedom to make decisions and use this platform.
For example, maybe we will build for the web, but Qt is not very suitable for the web because it is too large.

Reply:
OK, I see. I've also amended my #6588 to not use `QT_DEPRECATED` but reverted it to using `[[deprecated]]`, also because we now switched to C++17 anyway